src/crawler/crawlers/arso_daily_weather/crawler.js

```diff
@@ -0,0 +1,247 @@
+            rawMeasurements.push(text);
+            rawStyleHtml.push($(el).html());
+        });
+        $('.meteoSI-table thead tr th').each((i, el) => {
+            const text = $(el).text();
+            rawHeads.push(text);
+        });
+
+        const measurements = [];
+        while (rawMeasurements.length) measurements.push(rawMeasurements.splice(0, 11));
+
+        const heads = [];
+        while (rawHeads.length) heads.push(rawHeads.splice(0, 11));
+
+        const styleHtml = [];
+        while (rawStyleHtml.length) styleHtml.push(rawStyleHtml.splice(0, 11));
+
+        let i = 0;
+        for (const station of measurements) {
+            const stationData = {};
+            for (const varIdx in station) {
+                if (varIdx > 0) {
+                    let sensorName;
+                    if (heads[0][varIdx].includes('[')) {
+                        sensorName = heads[0][varIdx].split('[')[0].trim();
+                        stationData[sensorName] = station[varIdx];
+                    } else stationData[heads[0][varIdx]] = station[varIdx];
+                } else {
+                    stationData['Postaja'] = station[varIdx];
+                    stationData['Datum'] = heads[0][varIdx];
+                }
+
+                if (styleHtml[i][varIdx].includes('/uploads/meteo/style/img')) {
+                    // if image
+
+                    stationData[heads[0][varIdx]] = styleHtml[i][varIdx].slice(43, -6);
+                }
+            }
+            i++;
+
+
+            weatherData.push(stationData);
+            // console.log(stationData)
+        }
+
+        return weatherData;
+    };
+
+    stringDataConversion(measurement) {
+        if (measurement == measurement.toUpperCase()) {
+            // if measurement is WIND DIRECTION - TRUE DIRECTION
+            const windConversionTable = {
+                S: 180,
+                SE: 135,
+                SW: 225,
+                N: 0,
+                NE: 45,
+                NW: 315,
+                E: 90,
+                W: 270};
+            return (windConversionTable[measurement]);
+        } else {
+            // if measurement is WEATHER DESCRIPION
+            const descriptionConversionTable = {
+                clear: 0,
+                mostClear: 1,
+                partCloudy: 2,
+                modCloudy: 3,
+                modCloudy_lightRA: 4,
+                prevCloudy: 5,
+                overcast: 6,
+                overcast_lightRA: 7,
+                modRA: 8,
+                lightDZ: 9,
+                modDZ: 10,
+                modCloudy_lightDZ: 11
+            };
+            
+            if (descriptionConversionTable[measurement] == undefined) console.log("ARSO_DAILY_CRAWLER: DescriptionConversionTable missing value for weather description. Please update the table.") 
+            return (descriptionConversionTable[measurement]);
+        }
+    }
+
+
+    async dataLake(data) {
```

Comment:
Describe, what the function does, what is the data parameter. Use standard comments

```
/**
 * ...
 */
```